documentation/docs/14-configuration.md

```diff
@@ -148,7 +148,30 @@ See [Prerendering](#ssr-and-javascript-prerender). An object containing zero or
 
 - `crawl` — determines whether SvelteKit should find pages to prerender by following links from the seed page(s)
 - `enabled` — set to `false` to disable prerendering altogether
-- `force` — if `true`, a page that fails to render will _not_ cause the entire build to fail
+- `onError`
+
+  - `"fail"` — (default) fails the build when a routing error is encountered when following a link
+  - `"continue"` — allows the build to continue, despite routing errors
+  - function — you can use the `errorDetails` to determine whether or not to `throw`, thereby failing the build.
```

Comment:
I don't see anything defining `errorDetails`, so let's remove the reference to it
```suggestion
  - `function` — custom error handler allowing you to log, `throw` and fail the build, or take other action of your choosing based on the details of the crawl
```